src/main.cpp

```diff
@@ -215,6 +215,10 @@ loop() {
   MicroTask.update();
   Profile_End(MicroTask, 10);
 
+#if defined(NEO_PIXEL_PIN) && defined(NEO_PIXEL_LENGTH) && defined(ENABLE_WS2812FX)
+  ledManager_loop();
+#endif
```

Comment:
This should not be needed, the MicroTask and the LedManagerTask instance provide the equivelant without the need to add lots of loop calls to the main `loop` function.

If you can't use the LedManagerTask you could setup another task just for running the animations.

Reply:
Hi Jeremy. 
Understood. However the main reason to call this is to invoke the ws2812fx.service() method which refreshes the led status bar.

I'm not sure about the Microtasks architecture as I could find much information about.
The problem I have is that the service() method need to be invoked regularly to update the bar and If I do not force the call the bar is not updated. I could find any way within ledbartask to execute this regularly. Only when the EVSE changes status that invokes the service just once and then the animation is stopped

Should I call from main loop just a service method directly instead?  
I mean:
 create a method within the ledManagerTask  called service and then launch from the main loop? (that is in essence what this ledManager_loop call is doing)

Thanks in advance